src/components/pages/Login/LoginForm.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import uuid from 'uuid';
+import {
+  Container,
+  Button,
+  Grid,
+  Header,
+  Segment,
+  Form,
+  Image,
+  Message
+} from 'semantic-ui-react';
+import logo from '../../../assets/images/tiemendo_logo.jpg';
+
+const LoginForm = props => {
+  const { state, handleSubmit, handleChange } = props;
+
+  return (
+    <Container>
+      <Grid
+        textAlign="center"
+        style={{ height: '85vh' }}
+        verticalAlign="middle"
+      >
+        <Grid.Column style={{ maxWidth: 300 }}>
+          {state.errors.length && (
```

Comment:
Nice thinking on having the error modal be up top. Like the implementation as well!